charts/rancher-turtles/values.yaml

```diff
@@ -1,11 +1,14 @@
 rancherTurtles:
   image: controller
   imageVersion: v0.0.0
-  imagePullPolicy: Never
+  imagePullPolicy: IfNotPresent
   namespace: rancher-turtles-system
   managerArguments: []
   imagePullSecrets: []
   features:
+    cluster-api-operator:
+      cleanup: true
+      kubectlImage: bitnami/kubectl:1.27
```

Comment:
Do we want to align the pre-install `kubectl` image to this?

Reply:
I would use `registry.k8s.io/kubernetes/kubectl:v1.28.0` here, to rely on upsteam CI and not bitnami. Otherwise LGTM

Reply:
Alright, I decided to use `registry.k8s.io/kubernetes/kubectl:v1.28.0` image in both pre-install and post-delete hooks, and made changes accordingly.